### PR TITLE
 fix: learning rate scheduler fix for bert squad example [DET-2897]

### PR DESCRIPTION
**Pull request by davetroiano**
## Commentary (optional)

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: examples/experimental/trial/bert_squad_pytorch/const.yaml [hunk 1]
```diff
@@ -8,19 +8,26 @@ hyperparameters:
...
     num_warmup_steps: 0
-    num_training_steps: 400
     max_seq_length: 384
     doc_stride: 128
     max_query_length: 64
     n_best_size: 20
     max_answer_length: 30
     null_score_diff_threshold: 0.0
+    max_grad_norm: 1.0
+    num_training_steps: 15000 # This is the number of optimizer steps. Set it
```
Comment (davetroiano) [inline examples/experimental/trial/bert_squad_pytorch/const.yaml:18]: this is the main fix to achieve parity.  if this value is too low, [we end up with a LR of zero](https://github.com/huggingface/transformers/blob/eacea530c1ed63b182433a1b462d33881bd20409/src/transformers/optimization.py#L55-L57) pretty quickly, and this was causing the learning to basically stop (validation f1 flatlined at 60-something).

in the huggingface repo, they set this similarly to # batches times * epochs (assuming gradient accumulation is 1) -- see [here](https://github.com/huggingface/transformers/blob/master/examples/question-answering/run_squad.py#L87) and [here](https://github.com/huggingface/transformers/blob/master/examples/question-answering/run_squad.py#L100).

with this fix we get closer to the expected validation f1: 89 after 150 steps, compared to 88.52.

**Comment by davetroiano** [comment]
output from a const run:

image
